sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorTransmitter.cs

```diff
@@ -156,10 +161,13 @@ private ExportResult HandleFailures(HttpMessage httpMessage)
                 // HttpRequestException
                 content = HttpPipelineHelper.GetRequestContent(httpMessage.Request.Content);
                 result = _storage.SaveTelemetry(content, HttpPipelineHelper.MinimumRetryInterval);
+                AzureMonitorExporterEventSource.Log.Write($"FailedToTransmit{EventLevelSuffix.Informational}", "Network Error");
```

Comment:
Is this informational or Warning?

Reply:
We are saving telemetry so thought of it as informational message. But warning also makes sense. Should we change it in case of error response code as well?

Reply:
Yes. Let us change it to warning in case of error response too.